python/mxnet/gluon/estimator/event_handler.py

```diff
@@ -76,14 +76,20 @@ class LoggingHandler(EventHandler):
         file name to save the logs
     file_location: str
         file location to save the logs
+    verbose: int, default 1
+        Limit the display level of training progress
+        verbose=0: display nothing(silent)
```

Comment:
then why have this eventHandler?

Reply:
Removed the verbose `0` option as it makes no sense adding it. Thanks for your input